client/src/components/a.jsx

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+}
+
+body {
```

Comment:
Since our CSS is going to all be combined, you may want to name your css classes a little more specifically

Reply:
Is it ok to remove the body and * ?